.asf.yaml

```
@@ -21,6 +21,8 @@ github:
     squash: true
     rebase: true
     merge: true
+  protected_branches:
+    main
```

The referenced https://cwiki.apache.org/confluence/display/INFRA/git+-+.asf.yaml+features#Git.asf.yamlfeatures-BranchProtection is telling me

> All protected branches in the YAML must be dictionary entries. Thus, if you only want to disable force push from a branch, you can construct a fake dictionary like so:

```suggestion
    main: {}
```